demo/demo_director.py

```diff
@@ -742,8 +742,30 @@ def listen():
 
   server.register_function(clear_vehicle_targets, 'clear_vehicle_targets')
 
-  server.register_function(mitm_arbitrary_package_attack, 'mitm_arbitrary_package_attack')
-  server.register_function(undo_mitm_arbitrary_package_attack, 'undo_mitm_arbitrary_package_attack')
+  # Attack 1: Arbitrary Package, no keys
+  server.register_function(mitm_arbitrary_package_attack,
+      'mitm_arbitrary_package_attack')
+  server.register_function(undo_mitm_arbitrary_package_attack,
+      'undo_mitm_arbitrary_package_attack')
+
+  # Attack 2: Replay, no keys
+  server.register_function(prepare_replay_attack_nokeys,
```

Comment:
I guess so. Breaking it down:

The attack *here* only replays an old timestamp, and doesn't result in a firmware rollback. So _this attack_ is:
- a [Replay attack](https://en.wikipedia.org/wiki/Replay_attack) by common definition: because it replays previously used, otherwise valid metadata
- a [Rollback attack according to the TUF Security Doc](https://github.com/theupdateframework/tuf/blob/develop/SECURITY.md#attacks-and-weaknesses), which has a very general definition ("older **files** than those the client has already seen")
- NOT a [Rollback attack according to the TUF Spec itself](https://github.com/theupdateframework/tuf/blob/40aaf93f5a39f659cc312b66624f2cbff59b8582/docs/tuf-spec.txt#L166), because that refers to installing older software, which this does not do
- NOT a [Rollback attack according to the Uptane Design Document](https://docs.google.com/document/d/1pBK--40BCg_ofww4GES0weYFB6tZRedAjUy6PJ4Rgzk/edit#heading=h.4mo91b3dvcqd), which also defines rollback as installation of older software than the vehicle has.

Similar questions arise for the freeze attack label.

We don't really _have_ to address this (though it'd be nice to have a consistent definition). We could expand the attack here to perform a rollback of an installed image, with a little bit of effort.



 ("Freeze" and "rollback" are kinda from the perspective of the vehicle, as defined.